ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlTarget.java

```diff
@@ -118,7 +121,8 @@ public void postAsyncLagReportingRequest(
   ) {
     executeRequestAsync(
         LAG_REPORT_PATH,
-        lagReportingMessage
+        lagReportingMessage,
+        false /* expectFailures */
```

Comment:
We can rely on the IDE to annotate the argument instead of comment? :)

Reply:
Removed.